Neos.Flow/Classes/Aop/Builder/ProxyClassBuilder.php

```diff
@@ -114,12 +114,15 @@ public function injectReflectionService(ReflectionService $reflectionService)
     }
 
     /**
-     * @param SystemLoggerInterface $systemLogger
+     * Injects the (system) logger based on PSR-3.
+     *
+     * @param LoggerInterface $logger
      * @return void
+     * @Flow\Autowiring(false)
```

Comment:
Why is this done here, but not elsewhere?

Reply:
I think because some issues with this being used in compiletime.